framework/src/main/java/org/checkerframework/framework/type/TypeFromMemberVisitor.java

```diff
@@ -100,7 +101,8 @@ private static AnnotatedTypeMirror inferLambdaParamAnnotations(
             AnnotatedExecutableType functionType = f.getFunctionTypeFromTree(lambdaDecl);
             AnnotatedTypeMirror funcTypeParam = functionType.getParameterTypes().get(index);
             if (TreeUtils.isImplicitlyTypedLambda(declaredInTree)) {
-                if (f.types.isSubtype(funcTypeParam.actualType, lambdaParam.actualType)) {
+                if (TypesUtils.isErasedSubtype(
+                        funcTypeParam.actualType, lambdaParam.actualType, f.types)) {
                     // The Java types should be exactly the same, but because invocation type
```

Comment:
Could you update the comment to explain why the erased type is enough?

Could you add a test case that uses generic types and make sure that this doesn't introduce false negatives? E.g. have a method that accepts a `List<String>` and pass something that is a `List<@Nullable String>`. If this only compares the `List` part, it will ignore the mismatch in the type arguments.
If this isn't a concern, please document why.